about.html

```diff
@@ -0,0 +1,231 @@
+            <p>
+                Please contact us per Email for any further questions
+                about Kwita Izina 2022
+            </p>
+            <h6>
+                kwitaizina@gmail.com
+            </h6>
+        </div>
+    </div>
+
+        <!-- Creative commons -->
+    <div class="creative">
+        <div class="programtitle">
+            Creative commons best ceremony of the year
+        </div>
+        <div class="programline"></div>
+        <div class="program">
+            <p>
+                The ceremony is, first and foremost, an opportunity to thank the communities 
+                that live around the gorilla habitat.
+            </p>
+            <div class="imgicon">
+                <img src="image/kwitizina-logo.jpeg" alt="introduction icon">
+            </div>
+        </div>
+    </div>
+
+                    <!-- Past ceremony -->
+    <div class="pastceremony">
+        <div class="titleprogram">
+           <h3>See the Past Kwita Izina Cerermony</h3>
+        </div>
+        <div class="programline"></div>
+        <P>
+            Take look for the last two ceremonies of Kwita Izina.
+        </P>
+        <div class="lastceremony">
+            <div class="ceremonylast">
+                <img src="image/event-2019.jpg" alt="event 2019" class="event2019"/>
+                <div class="pastyear">
+                    <h1>2019</h1>
+                </div>
+                <div class="pasttext">
+                    <p>
+                        Kwita izina 2019 in Musanze Kigini
+                    </p>
+                </div>
+            </div>
+            <div class="ceremonylast">
+                <img src="image/event-2020.jpg" alt="event 2020" class="event2019"/>
+                <div class="pastyear">
+                    <h1>2020</h1>
+                </div>
+                <div class="pasttext">
+                    <p>
+                        Kwita izina 2019 in Musanze Kigini
+                    </p>
+                </div>
+            </div>
+        </div>
+    </div>
+                <!-- partner side -->
+    <section class="partner">
+        <div class="partnertitle">
+            <h2>Partner</h2>
+        </div>
+        <div class="partnerline"></div>
+        <div class="partnerlogo">
+           <div class="partnerimg">
+               <img src="image/government.png" alt="government of rwanda seal" class="rwandalogo"/>
+           </div>
+           <div class="partnerimg">
+               <img src="image/logo-rdb.jpg" alt="RDB logo" class="logordb"/>
+           </div>
+            <div class="partnerimg">
+                <img src="image/visitirwandaLogo.png" alt="visit rwanda logo" class="vistirwanda"/>
+            </div>
+            <div class="partnerimg">
+                <img src="image/rema.png" alt="rema logo" class="remalogo">
+            </div>
+        </div>
+                    
+    </section>
+
+                <!-- copyright -->
+    <div class="footer">
```

Comment:
- [ ] [REQUIRED] Please, use `footer` instead of `div` for semantic purposes ♻️.

Reply:
Done